src/admin/pages/Premium.vue

```diff
@@ -13,6 +13,20 @@
         </div>
         <!-- End of Header Section -->
 
+        <!--   Vendor Capabilities Banner     -->
+        <div class="vendor-capabilities-banner" :style="bannerBg">
```

Comment:
This section is almost the same as VendorCapabilities.vue's banner section. Can we do one thing to put this in a component and add props for the banner image. Also, we can add a slot so that if we need we can add buttons too.